src/devices/owon.ts

```diff
@@ -647,8 +1130,9 @@ export const definitions: DefinitionWithExtend[] = [
         model: "SLC631",
         vendor: "OWON",
         description: "Smart plug with doorbell press indicator",
+        meta: {multiEndpoint: true},
         extend: [
-            m.onOff({endpointNames: ["l1", "l2", "l3"]}),
+            m.onOff({endpointNames: ["l1", "l2", "l3"], powerOnBehavior: false}),
             m.iasZoneAlarm({
```

Comment:
Use `m.deviceEndpoints` here (remove `meta: {multiEndpoint: true},` and `endpoint: (device) => ({l1: 1, l2: 2, l3: 3}),`)